src/OAuth2Login.jsx

```diff
@@ -82,15 +82,19 @@ class OAuth2Login extends Component {
   }
 
   render() {
-    const { className, buttonText, children } = this.props;
-    const attrs = { onClick: this.onBtnClick };
+    const { className, buttonText, children, render } = this.props;
 
-    if (className) {
-      attrs.className = className;
+    if (render) {
+        return render({className, buttonText, children, onClick: this.onBtnClick});
```

Comment:
I'd put a space after the opening brace & before the closing one (sorry for being picky 😅 )

Reply:
Not a problem! We all have our styles. Fixed.